python/CMakeLists.txt

```diff
@@ -0,0 +1,118 @@
+find_package(pybind11 2.5.0 REQUIRED)
+
+if ("${PYTHON_INSTALL_TARGET}" STREQUAL "")
+	message(STATUS "PYTHON_INSTALL_TARGET variable is not set, setting to a default one...")
+	execute_process(COMMAND "python3" "-c" "import site; print(site.getsitepackages()[0])"
+					OUTPUT_VARIABLE PYTHON_INSTALL_TARGET OUTPUT_STRIP_TRAILING_WHITESPACE)
+endif ()
+
+message(STATUS "The Python module will be install at this location : ${PYTHON_INSTALL_TARGET}")
+
+set(PYBIND11_SOURCES
+	#pointmatcher module
+	pointmatcher/datapoints.cpp
+	pointmatcher/datapointsfilter.cpp
+	pointmatcher/datapointsfilters.cpp
+	pointmatcher/errorminimizer.cpp
+	pointmatcher/icp.cpp
+	pointmatcher/icpchainbase.cpp
+	pointmatcher/icpsequence.cpp
+	pointmatcher/impls/inspectors_impl.cpp
+	pointmatcher/impls/matchers_impl.cpp
+	pointmatcher/impls/outlierfilters_impl.cpp
+	pointmatcher/impls/transformations_impl.cpp
+	pointmatcher/impls/transformationcheckers_impl.cpp
+	pointmatcher/impl.cpp
+	pointmatcher/inspector.cpp
+	pointmatcher/io.cpp
+	pointmatcher/matcher.cpp
+	pointmatcher/matches.cpp
+	pointmatcher/outlierfilter.cpp
+	pointmatcher/outlierfilters.cpp
+	pointmatcher/pointmatcher.cpp
+	pointmatcher/transformation.cpp
+	pointmatcher/transformations.cpp
+	pointmatcher/transformationchecker.cpp
+	pointmatcher/transformationcheckers.cpp
```

Comment:
Why the change in capitalization of source files?

Reply:
These are the source files for the bindings only. Since it was my first time making python bindings for a C++ library, I found these [guidelines](https://developer.lsst.io/v/DM-8980/pybind11/style.html) and tired to follow some of them. But now, I realize that I misinterpreted [that guideline](https://developer.lsst.io/v/DM-8980/pybind11/style.html#wrappers-for-a-c-header-file-should-go-in-a-python-module-with-a-lowercased-version-of-the-header-file-name) that was suggesting to use lowercase module and not lowercase source files. I thought it was to minimize ambiguity between the real source files and the wrapper source files. I will eventually rename them with the right convention. 😅